### ISSUE TITLE
md-input input element color not affected by parent.

### ISSUE THREAD

**Issue by DennisSmolek** [angular/material2#534]
#### Bug:
When using template syntax to change the md-input text color with `[style.color]="whatever"` it does not affect the resulting `<input>` element.

I found this when working on a color picker element and I tried changing the input's text color to match the chosen color.

#### What is the expected behavior?
The input element should change its text color/properties based on changes to md-input

#### What is the current behavior?
Default values (like Plunker/Chrome's user agent stylesheet's initial value) override the div's color change.

#### What are the steps to reproduce?
```
<md-input   [style.color]="color2"   [value]="color2"></md-input>
```

Link to in action and a fix.
https://plnkr.co/edit/epsQlY?p=preview

To fix it the `.md-input-element` class needs the property: `color:inherit`

With this I assume there is a few other css properties that need to be checked..

#### Which versions of Angular, Material, OS, browsers are affected?
Angular: RC.1 Material: Latest.. Only tested on Mac/Chrome at the moment but Its an easy fix..

**Comment by DennisSmolek** [comment]
Another related inheritance issue: #277

**Comment by jelbourn** [comment]
@hansl I'd lean towards this being the user sets via css on the `.md-input-element`. Thoughts?

**Comment by jelbourn** [comment]
I believe this was fixed in #1262
